assets/setup.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
Can we un-CIQSify the solution entirely?
By that I mean, abandon 'assets', 'core', 'src' and instead simply put the scripts and ARM templates somewhere.
src is probably the right place for that.